### DISCUSSION TITLE
Drasil Research Group Meeting Agenda, 1:30 pm, Friday Mar 4, 2022 (On Teams)

### DISCUSSION THREAD

**Discussion by smiths** [JacquesCarette/Drasil#2949] [Meeting Agendas]
The agenda for this coming week is as follows:

0. Update on summer employees - not likely to be making any hires this summer.  😞 
1. Jason update
2. Peter update
3. Dong update
4. Cold cases:
a. (from #2945) @balacij https://github.com/JacquesCarette/Drasil/issues/2853#issuecomment-1034427454 + https://github.com/JacquesCarette/Drasil/issues/2911#issuecomment-1049180439
b. @peter-michalski #135 
c. @cd155 #492
5. Questions/Comments/Concerns/Discussion

**Comment by smiths** [comment]
@peter-michalski and @cd155, please add new issues to next week's agenda when you get a chance.

**Comment by balacij** [comment]
Regarding the cold case I had up there, we ended off on a question: "Should properties be used to describe knowledge fragment types?" e.g., typeclasses = properties, instances of typeclasses = proofs, records + list of properties = types declarations, data + type signatures = instances of captured knowledge.

**Comment by cd155** [comment]
updated the cold case

**Comment by peter-michalski** [comment]
added cold case

**Comment by smiths** [comment]
I've moved the code case issue from @peter-michalski to next week's agenda.  I've also kept the issue from @cd155 for this week in next week's agenda, so that we can get some feedback from @JacquesCarette.
